device-types/FS/S5860-20SQ.yaml

```diff
@@ -0,0 +1,73 @@
+---
+manufacturer: FS
+model: S5860-20SQ
+slug: s5860-20sq
+part_number: s5860-20sq
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: Management
+    type: rj-45
+power-ports:
```

Comment:
It looks like the PSUs are modular. These should be modelled as `module-bays` and then you should add the applicable PSUs for the device as module-types if they are not already created

It looks like the smart fans are also modular, so please create them in the same fashion

Reply:
Thanks Daniel. From what I see this should apply to the other FS switches too, I would be happy to fix them.

I guess this should cover it.
![image](https://user-images.githubusercontent.com/30608412/223843743-336bd422-3087-4fac-bb36-afeec9c31801.png)
![image](https://user-images.githubusercontent.com/30608412/223843832-9f717f7f-b0ee-4ffe-afb3-5430f22b4475.png)

```
module-bays:
  - name: PSU1
    label: PSU1
    position: PSU1
  - name: PSU2
    label: PSU2
    position: PSU2
  - name: FAN1
    label: FAN1
    position: FAN1
  - name: FAN2
    label: FAN2
    position: FAN2
```

Reply:
Absolutely, these should apply to the other devices as well. A lot of these were created before this was available, or they were just mistakenly created that way, so we have to update them as we get PRs like these.

If you would be willing to update the others that would be a **HUGE** help! I would majorly appreciate it!

Reply:
Sure, it helps me too. I fixed this for this model and added the specific PSU on module-types at module-types/FS/SPSM-C150WAC.yaml
LMK if that should be the proper way and I'll cover the rest soon.

Reply:
Ah interesting, makes sense to me. I just fixed it.